lisa_commands/request_helper.py

```diff
@@ -147,7 +147,7 @@ def youtube_search(query):
         'part': 'snippet',
         'q': query,
         'key': GOOGLE_API_KEY,
-        'maxResults': 10
+        'maxResults': 3
```

Comment:
Are we sure we don't want to put youtube to first result only? In almost all `youtube me` cases I can think of, I feel like people generally want to find a specific video. It's not quite like `image me` where there can be multiple acceptable images, so it just needs to grab any one of them.

Reply:
I don't have a super strong objection to that. If no one else weighs in then I'll do that.